sdk/samples/keyvault/keyvault/samples/src/keys_client_example.c

```diff
@@ -65,17 +77,22 @@
       printf("\n"); \
       printf(message); \
       printf("\n"); \
-      return 1; \
+      EXIT(1); \
     } \
   } while (0)
 
-int exit_code = 0;
-
 az_span get_key_version(az_http_response* response);
 az_span const key_name_for_test = AZ_SPAN_LITERAL_FROM_STR("test-new-key");
 
 int main()
 {
+
+// If running with libcurl, call global init. See project Readme for more info
+#ifdef TRANSPORT_CURL
+  TEST_FAIL_ON_ERROR(curl_global_init(CURL_GLOBAL_ALL), "Fail to init libcurl");
```

Comment:
You can't pass the result of `curl_global_init()` to `TEST_FAIL_ON_ERROR`, because it is not az_result.

Reply:
My suggestion no longer applies. Love the new way we have it now!